tests/testthat/test-dm-from-src.R

```diff
@@ -8,7 +8,8 @@ test_that("table identifiers are quoted", {
     map_chr(dbplyr::remote_name)
 
   con <- dm_get_con(dm)
-  expect_equal(unname(remote_names), unclass(DBI::dbQuoteIdentifier(con, names(dm))))
+  # `gsub()`, cause schema names are part of the remote_names (also standard schemas "dbo" for MSSQL and "public" for Postgres).
+  expect_equal(gsub("^.*\\.", "", unname(remote_names)), unclass(DBI::dbQuoteIdentifier(con, names(dm))))
```

Comment:
I wonder how this test could even succeed before?

Reply:
It does fail locally on Postgres for me. Anyway...